ecu-sw-bootloader/Core/Inc/blt_conf.h

```diff
@@ -0,0 +1,62 @@
+#ifndef BLT_CONF_H
+#define BLT_CONF_H
+
+/****************************************************************************************
+*   C P U   D R I V E R   C O N F I G U R A T I O N
+****************************************************************************************/
+/* External crystal frequency (if you only use HSI = 8 MHz, just leave at 8000). */
+#define BOOT_CPU_XTAL_SPEED_KHZ (16000)
+
+/* System clock frequency */
+#define BOOT_CPU_SYSTEM_SPEED_KHZ (128000)
+
+/* STM32C0 is little endian. */
+#define BOOT_CPU_BYTE_ORDER_MOTOROLA (0)
+
+/* Optional hook before jumping to user application. */
+#define BOOT_CPU_USER_PROGRAM_START_HOOK (0)
+
+/****************************************************************************************
+*   C O M M U N I C A T I O N   I N T E R F A C E   C O N F I G U R A T I O N
+****************************************************************************************/
+/* CAN communication */
+#define BOOT_COM_CAN_ENABLE (1)
+#define BOOT_COM_CAN_BAUDRATE (500000) /* pick what matches your bus */
```

Comment:
Baudrate should be 1M
```suggestion
#define BOOT_COM_CAN_BAUDRATE (1000000) /* pick what matches your bus */
```

Reply:
required also modification of ioc file